backend/blueprint/api/utils/response.py

```diff
@@ -12,8 +14,12 @@ def generateMessage(self, message):
     def generateURL(self, url, message):
         self.message = message
         self.URL = url
+        self.sanitizeURL()
         return self.serialize()
```

Comment:
shift `self.serialize()` inside `self.sanitizeURL()` maybe?
:sweat_smile:  what say?

Reply:
maybe I am talking about something like 
```python
return self.saniitizeURL()
```

where sanitize has declaration as:
```python
def sanitizeURL(self):
    self.URL = #change the URL
    return self.serialize()
```

But it is okay, your approach is great. I was just suggesting a minor change :sweat_smile: :+1:

Reply:
Sure that i can do :+1: